frontend/main.js

```diff
@@ -374,7 +375,12 @@ async function startApp () {
       }).catch(e => {
         console.error(`[main] caught ${e?.name} while fetching settings or handling a login error: ${e?.message || e}`, e)
       }).finally(() => {
-        Vue.set(this.ephemeral, 'ready', true)
+        this.ephemeral.ready = true
+
+        const loadingScreenEl = document.querySelector('#main-loading-screen')
+        if (loadingScreenEl) {
+          loadingScreenEl.classList.add('is-removed')
+        }
```

Comment:
Why couldn't the component be removed or its display set to `none`? I think a clarifying comment would be useful here

Reply:
@snowteamer I think it makes sense to remove it completely instead of hiding it if there is a case that the app needs to display this loader screen.